src/batch/batch.ts

```diff
@@ -24,9 +23,16 @@ export class Batch {
   ): Promise<CreateBatchResponse> {
     for (const email of payload) {
       if (email.react) {
-        email.html = await renderAsync(email.react as React.ReactElement);
-        // biome-ignore lint/performance/noDelete: <explanation>
-        delete email.react;
+        try {
+          const { renderAsync } = await import('@react-email/render');
```

Comment:
Would it be worth doing the try-catch around only the `await import`? Might get confusing if the `renderAsync` call itself fails